proof/dleq.go

```diff
@@ -0,0 +1,109 @@
+package proof
+
+import (
+	"errors"
+
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/hash"
+	"github.com/dedis/crypto/random"
+)
+
+// This package provides functionality to create and verify non-interactive
+// zero-knowledge (NIZK) proofs for the equality (EQ) of discrete logarithms (DL).
```

Comment:
doc: Please add some more info about what you're doing here, with a short 'resume' of the algorithm used.